data/human/free worlds 0 prologue.txt

```diff
@@ -808,25 +802,27 @@ event "fw escorts abandon timer"
 
 mission "FW Escort 2"
 	autosave
+	landing
 	name "Free Worlds Escort"
 	description "Escort a Free Worlds supply convoy safely to <destination>."
 	source "Longjump"
 	destination "Wayfarer"
 	to offer
 		has "FW Escort 1B: done"
-		not "event: fw escorts abandon timer"
 	to fail
 		has "chosen sides"
 	
 	on offer
 		conversation
-			`A few of the escort pilots, yourself included, have stuck around for another mission, and luckily a few new faces have joined in as well. Tomek meets you in the spaceport and says, "You know the drill. Escort the freighters to <planet> and back. 400,000 credits for your troubles. Plus, please believe that all of you are known to the Council, and your bravery will not be forgotten. We can't afford to lose any of these freighters. Who's willing to help escort them?"`
+			`Tomek is waiting in <origin>'s spaceport to receive the freighters. After double-checking that the cargo is all intact, he turns to you and the rest of the escort captains and says, "All of you have done a great service for the Free Worlds." He walks over and personally presents a credit chip to each captain.`
+			`	After handing over your payment, Tomek announces, "The convoy is heading back out for <planet> immediately. Same deal as before: 400,000 credits for the safe return of the freighters, their crew, and their cargo. And I swear your courage will never be forgotten by the Council or the Free Worlds. If you wish to volunteer a second time, please stay here."`
```

Comment:
"If you wish" sounds like volunteering is personally beneficial to you.

Also, "back out" is a bit weird since you're heading from one industrial centre to another, right?

```suggestion
			`	After handing over your payment, Tomek announces, "The convoy is heading back to <planet> immediately. Same deal as before: 400,000 credits for the safe return of the freighters, their crew, and their cargo. And I swear your courage will never be forgotten by the Council or the Free Worlds. If you're willing to volunteer a second time, please stay here."`
```